packages/tools/client-debug-view/src/components/data-object-views/SharedCounterView.tsx

```diff
@@ -37,12 +54,65 @@ export function SharedCounterView(props: SharedCounterViewProps): React.ReactEle
 		};
 	}, [sharedCounter, setValue]);
 
+
+    function decrementCounter(): void {
+        sharedCounter.increment(-deltaValue);
+	}
+
+    function incrementCounter(): void {
+		sharedCounter.increment(deltaValue);
+	}
+
+    const inputSetValue = (e: React.ChangeEvent<HTMLInputElement>): void => {
+        numberFormatCheck(e.target.value);
+    }
+
+    const numberFormatCheck = (val: string): void => {
+        const num = Number.parseInt(val, 10) || value;
```

Comment:
What does the "|| value" do here?

Reply:
I wrote that clause in case `val` is `undefined`, and if so replace with the existing `value`.

Reply:
Ah, okay. `value` is the value of the counter though. I think this needs to be `deltaValue`?

Does this mean that the user cannot clear the form to 0 by hitting backspace? I think this will mean the user can't hit backspace on a single-digit value.